bin/node/runtime/src/lib.rs

```diff
@@ -416,7 +416,7 @@ impl pallet_babe::Config for Runtime {
 	type DisabledValidators = Session;
 	type WeightInfo = ();
 	type MaxAuthorities = MaxAuthorities;
-	type MaxNominators = MaxNominatorRewardedPerValidator;
+	type MaxNominators = MaxExposurePageSize;
```

Comment:
This is used for weight calculation for reporting equivocation. The older configuration (changed [here](https://github.com/paritytech/substrate/pull/14471)) was incorrect since for slashing, the max nominators exposed could be larger than `MaxNominatorRewardedPerValidator `. `MaxNominatorRewardedPerValidator` is only used to limit nominators that can be rewarded but nominators eligible for slashing (per validator) is unbounded (though there is a hard count for all nominators => 22500).  

Not sure the correct way to solve this and is most probably out of scope of this PR. Setting this to `MaxExposurePageSize` is strictly same as before (though now its more apparent that this configuration is wrong).